<launch>

<node name="otimizar_nuvem" pkg="laser_camera" type="otimizar_nuvem_node" output="screen">
    <param name="nome_nuvem"   value="/home/vinicius/Desktop/Dados_B9/nuvem_final.ply"/>
    <param name="mean_or"      value="1"   />
    <param name="deviation_or" value="0.6" />
    <param name="degree_p"     value="2"   />
    <param name="radius_p"     value="0.05"/>
</node>

</launch>
